r4e-aarch-image.sh

```diff
@@ -0,0 +1,80 @@
+# script for creating an image that points to a certain ostree commit from web server
+# the image includes the agent RPM that communicates with server that monitors edge devices
+# at the end the following products will be available for download:
+#   - ostree image
+#   - kickstart file
+# input:
+#  - ip or hostname with port if different than 80 used for connecting to the local web server
+#  - a name for the image. allows using different ostree image for different types of edge devices
+#  - URL for connecting to the server that monitors edge devices
+# script assumes that files `aarch64-blueprint.template` and `ks.cfg.template` are present in the working directory
+set -e
+Usage() {
+    echo "Usage:"
+    echo `basename $0` "<image-server-address> <image-name> <agent-server-url>"
+}
+
+# check usage
+if [ $# != 3 ] ; then
+    echo -e "bad usage\n"
+    Usage
+    exit 1
+fi
+
+# read input
+IMAGE_HOST=$1
+export IMAGE_NAME=$2
+export AGENT_URL=$3
+export OS_NAME="rhel-edge"
```

Comment:
the x86 script has different OS_NAME and REMOTE_OS_NAME. Can u explain the difference?

Reply:
@masayag @pkliczewski please see my comment

Reply:
REMOTE_OS_NAME is retrieved by `ostree remote list`, `rhel-edge` is the value returned by it for the image created for aarch64.
OS_NAME is taken from the deployed device, which in case of aarch64 is `rhel-edge` as well.

I haven't tested x86 image. How did you pick those values for the script you wrote ?